programs/ziti-edge-tunnel/package/deb/postrm.in

```diff
@@ -13,7 +13,13 @@ if [ "$1" = "purge" ]; then
 fi
 # End copied section
 
-if [ -L @SYSTEMD_UNIT_DIR@/@SYSTEMD_UNIT_FILE_NAME@ ]; then
```

Comment:
An earlier version of the package created a symlink at this location, but recent versions always place a regular file copied from the package's "share" directory, clobbering anything at this location. This means there's no possibility that the remove operation needs to remove a symlink at this path.